protocol-designer/src/steplist/formLevel/handleFormChange/dependentFieldsUpdateMix.ts

```diff
@@ -164,6 +164,23 @@ const updatePatchOnTiprackChange = (
   return patch
 }
 
+const updatePatchOnNozzleChange = (
+  patch: FormPatch,
+  rawForm: FormData,
+  pipetteEntities: PipetteEntities
+): FormPatch => {
+  if (
+    Object.values(pipetteEntities).find(pip => pip.spec.channels === 96) &&
+    fieldHasChanged(rawForm, patch, 'nozzles')
+  ) {
+    return {
+      ...patch,
+      ...getDefaultFields('aspirate_wells', 'dispense_wells'),
```

Comment:
ah sorry forgot to mention that mix has different fields. I think its just `wells`

```suggestion
      ...getDefaultFields('wells'),
```